x-pack/plugins/fleet/server/saved_objects/migrations/synthetics/to_v8_8_0.ts

```diff
@@ -66,6 +68,12 @@ export const migratePackagePolicyToV880: SavedObjectMigrationFn<PackagePolicy, P
     }
   }
 
+  // set location_id.id to agentPolicyId
+  if (enabledStream.vars && enabledStream.vars.location_id) {
+    enabledStream.vars.location_id.value = agentPolicyId;
```

Comment:
So this var doesn't yet exist, but will in the version of the policy that will be shipped with 8.8.0. Because of that, we may want to also add in the type? 

```
    enabledStream.vars.location_id = {
        type: 'text',
        value: agentPolicyId
    };
```
I'm not sure if creating a brand new variable that doesn't yet exist causes issues, @kpollich